### PR TITLE
[bitnami/wordpress] Remove updateStrategy.rollingUpdate: {} from wordpress chart to allow using Recreate update strategy

### PR DESCRIPTION
**Pull request by andreas-eberle**
### Description of the change

Currently, the wordpress helm chart specifies the following in the values.yaml

```
updateStrategy:
  type: RollingUpdate
  rollingUpdate: {}
```

When you use the wordpress helm chart as a subchart of another helm chart, there is no way to overwrite the `updateStrategy.rollingUpdate` property to remove it. This causes problems as soon as you want to use the `Recreate` update strategy. Then helm throws the following error:

```
Deployment.apps "my-app-wordpress" is invalid: spec.strategy.rollingUpdate: Forbidden: may not be specified when strategy `type` is 'Recreate'
```

To fix, this, I align the values.yaml with that of other bitnami helm charts like Kibana: https://github.com/andreas-eberle/charts/blob/main/bitnami/kibana/values.yaml#L104

### Benefits

With this change, it becomes possible to use the wordpress helm chart as a subchart with the Recreate update strategy. This is needed if you don't have WriteMany volumes. 

### Possible drawbacks

none

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/wordpress/values.schema.json [hunk 1]
```diff
@@ -1,212 +1,1659 @@
...
-          "form": true,
-          "description": "Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database switch this off and configure the external database details"
-        },
-        "primary": {
-          "type": "object",
-          "properties": {
-            "persistence": {
-              "type": "object",
-              "properties": {
+    "title": "Chart Values",
```
[the remark this answers is not in the document]
Comment (fmulero) [reply bitnami/wordpress/values.schema.json:2]: Thanks a lot for taking care about this. That file is not updated and it is following a previous standard. We have to review all the existing ones. Could you revert the changes in this file?

[the remark this answers is not in the document]
Comment (andreas-eberle) [reply bitnami/wordpress/values.schema.json]: Sure @fmulero. I removed these changes again.

**Review by dgomezleon — APPROVED** [review]
LGTM

**Comment by andreas-eberle** [comment]
Thanks for the fast review and merge!

**Comment by andreas-eberle** [comment]
Hey @dgomezleon , 

I saw that the CD job for this merge failed with some linting crash. Do you know what's wrong there? Is there something I need to fix or do you just need to retrigger the build?

https://github.com/bitnami/charts/actions/runs/4231562901

Thanks for checking!

**Comment by dgomezleon** [comment]
Hi @andreas-eberle ,

Re-running it made it work.

Thanks

**Pull request merged**